.gitignore

```diff
@@ -1,2 +1,2 @@
 /target
-.vscode/settings.json
+.vscode/launch.json
```

Comment:
Why aren't we ignoring vscode settings anymore?

Reply:
Because I was supposed to check in a few things here - but forgot.
E.g like using clippy for checkonsave vs. cargo check